### ISSUE TITLE
ask confirmation before remoing entries or groups.

### ISSUE THREAD

**Issue by zveber** [Kunzisoft/KeePassDX#947]
It would be much convinient if the app will ask confirmation of removing entries or groups showing list of their names to prevent acsident removing.

**Comment by Ilithy** [comment]
I agree, this protects against false manipulation, similarly, it would be safer to also ask for confirmation before removing the advanced unlocking key.
